### DISCUSSION TITLE
Custom background overflowing the view

### DISCUSSION THREAD

**Discussion by alexandre-g** [LucasMucGH/BottomSheet#115] [Q&A]
Hi there,

I am using the customBackground builder with an image like so:
```
.customBackground {
    Image("dots")
        .resizable()
        .aspectRatio(contentMode: .fill)
}
```

The image overflows the bottom sheet.
Any ideas on what's the best way to handle?
Using .fit keeps the correct height but does not fill the sides as it's a tall image

**Comment by lucaszischka** [comment]
You could try using clipped.

**Reply by alexandre-g** [reply]
Thanks for the suggestion but didn't work.
Below code is working fine for me inside the hosted body.. but if I wanted to use the close button that wouldn't be enough
```
.background(
    Image("dots")
        .resizable()
        .aspectRatio(contentMode: .fill)
)
```

**Reply by lucaszischka** [reply]
Maybe try to embed it in a V or HStack, make use of ZStacks or even set the height via the frame manually. Or if the code snipped you just shared works, try to add it to a Color.clear inside of the .customBackground.
